english/iot/scan-network/scan-network.md

```diff
@@ -5,3 +5,13 @@ His core temp starts to rise as his panic and extra clock cycles start to take e
 How is this possible!
 His clock cycles max out tyring to comprehend the situation.
 "Whats happened to the network!? Who is -1?" 
+
+Suddenly a Raspberry stumbles in with a Pi on his ARM. Raspberry yawns and blinks tiredly at 192.168.100.1. 
+"Thanks for the 2's compliment but broadcasting around here is considered impolite.....and you woke me to boot which I am less than grateful for".
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.